client/my-sites/stats/all-time-views-section/style.scss

```diff
@@ -0,0 +1,152 @@
+@import "@automattic/typography/styles/variables";
+@import "@wordpress/base-styles/breakpoints";
+
+$header-font: Recoleta, $sans;
+$mobile-layout-breakpoint: $break-small;
+$common-border-radius: 4px;
+
+.stats__all-time-views-section {
+	.highlight-cards {
+		background-color: inherit;
+		box-shadow: none;
+
+		@media ( max-width: $mobile-layout-breakpoint ) {
+			display: none;
+		}
+	}
+
+	.highlight-cards-list .highlight-card {
+		padding: 24px;
+		min-width: 320px;
+	}
+
+	.highlight-card-heading {
+		font-family: inherit;
+		font-weight: 500;
+		font-size: $font-title-small;
+		line-height: 26px;
+		color: var(--color-neutral-100);
+		margin-bottom: 16px;
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+	}
+
+	.segmented-control {
+		.segmented-control__link {
+			padding: 10px 12px;
+			font-weight: 500;
+			line-height: 20px;
+			color: #000;
+
+			&:focus {
+				outline: none;
+			}
+		}
+
+		.segmented-control__item {
+			&.is-selected {
+				.segmented-control__link {
+					background-color: var(--color-primary);
+					border-color: var(--color-primary);
+					color: var(--studio-white);
+				}
+			}
+
+			&:first-of-type .segmented-control__link {
+				border-top-left-radius: $common-border-radius;
+				border-bottom-left-radius: $common-border-radius;
+				border-right: 0;
+			}
+
+			&:last-of-type .segmented-control__link {
+				border-top-right-radius: $common-border-radius;
+				border-bottom-right-radius: $common-border-radius;
+				border-left: 0;
+			}
+		}
+	}
+
+	table.stats-views__months {
+		table-layout: fixed;
```

Comment:
I'd like to give the year column and totals column a little extra room. Removing `table-layout: fixed;` in the inspector solved the problem, but I'm unfamiliar with this declaration. Is this necessary? If not can we remove it to adjust those widths?

Reply:
Yes, the width of cells would be calculated automatically without setting the `table-layout: fixed`. I was trying to make the table toggling look smooth.